web/pdf_viewer.js

```diff
@@ -274,8 +272,13 @@ class PDFViewer {
     this.enablePrintAutoRotate = options.enablePrintAutoRotate || false;
     if (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) {
       this.removePageBorders = options.removePageBorders || false;
+
+      if (options.useOnlyCssZoom) {
+        console.error(
```

Comment:
Unless I'm mistaken this immediately makes the `useOnlyCssZooom` option unavailable. If so, I think we should deprecate this first, even if it's only for one release, so any users can switch gracefully without this being unavailable immediately. We can probably just translate the option to `maxCanvasPixels = 0` here and raise a deprecation warning if we do.